front/src/applications/editor/tools/rangeEdition/components/RangeEditionTrackRangeList.tsx

```diff
@@ -30,48 +33,43 @@ const TrackRangesList = () => {
   const ranges = entity.properties.track_ranges || [];
   const displayedRanges = showAll ? ranges : ranges.slice(0, DEFAULT_DISPLAYED_RANGES_COUNT);
 
-  const rangesByRoute = groupBy(displayedRanges, (range) => {
-    const isCurrentRange = compareTrackRange(range);
-    // eslint-disable-next-line no-restricted-syntax
-    for (const [routeKey, { trackRanges }] of Object.entries(routeElements)) {
-      if (isCurrentRange(routeExtra[routeKey]) || trackRanges.some(isCurrentRange)) {
-        return routeKey;
-      }
+  function makeFullTrackList() {
+    if (withRouteName) {
+      const rangesByRoute = groupBy(displayedRanges, (range) => {
+        const isCurrentRange = compareTrackRange(range);
+        // eslint-disable-next-line no-restricted-syntax
+        for (const [routeKey, { trackRanges }] of Object.entries(routeElements)) {
+          if (isCurrentRange(routeExtra[routeKey]) || trackRanges.some(isCurrentRange)) {
+            return routeKey;
+          }
+        }
+        return undefined;
+      });
+      return Object.entries(rangesByRoute).reduce<JSX.Element[]>((acc, [route, tracks], index) => {
```

Comment:
Use range over track, otherwise we can get confused with trackSection
```suggestion
      return Object.entries(rangesByRoute).reduce<JSX.Element[]>((acc, [route, ranges], index) => {
```

Reply:
done